CriticalMass/TextFieldWithInsets.swift

```diff
@@ -57,16 +57,6 @@ class TextFieldWithInsets: UITextField {
         return didBecomeFirstResponder
     }
 
-    // sets the backgroundColor when input ends
-    // kind of a hack sice the textField was always resetting its backgrundColor when it resigned firstResponder
-    override func resignFirstResponder() -> Bool {
```

Comment:
@mltbnz Is that save to remove? I don't see any issues without that hack and it fixes the missing placeholder after resigning first responder

Reply:
You are right. Works also without it.

Reply:
Then it is save :) Was there because of the weird way the TextField was acting with the theme change